code/modules/reagents/newchem/toxins.dm

```diff
@@ -175,17 +180,12 @@ datum/reagent/cyanide/on_mob_life(var/mob/living/M as mob)
 datum/reagent/itching_powder
 	name = "Itching Powder"
 	id = "itching_powder"
-	description = "Lots of annoying random effects, chances to do some brute damage from scratching. 6% chance to decay into 1-3 units of histamine."
+	description = "An abrasive powder beloved by cruel pranksters."
 	reagent_state = LIQUID
-	color = "#CF3600"
+	color = "#B0B0B0"
 	metabolization_rate = 0.3
 	penetrates_skin = 1
 
-/datum/reagent/itching_powder/reaction_mob(var/mob/living/M, var/method=TOUCH, var/volume)
-	if(method == TOUCH)
-		M.reagents.add_reagent("itching_powder", volume)
```

Comment:
we have penetrates_skin now so splashing it on someone will transfer the reagent to the mob itself--which has the effect you intend.